ArduCopter/land_detector.cpp

```diff
@@ -174,10 +190,16 @@ void Copter::update_throttle_thr_mix()
         // check for requested decent
         bool descent_not_demanded = pos_control.get_desired_velocity().z >= 0.0f;
 
-        if ( large_angle_request || large_angle_error || accel_moving || descent_not_demanded) {
-            attitude_control.set_throttle_mix_max();
-        } else {
+        bool xy_speed_low = (position_ok() || optflow_position_ok()) && xy_speed_cms <= 125.0f;
+        bool xy_speed_demand_low = pos_control.is_active_xy() && xy_des_speed_cms <= 125.0f;
+        bool slow_descent_demanded = pos_control.is_active_z() && des_climb_rate_cms < 0.0f && des_climb_rate_cms >= -100.0f;
+
+        bool slow_auto_descent = xy_speed_demand_low && xy_speed_low && slow_descent_demanded;
```

Comment:
slow_auto_descent will be true when vehicle is holding altitude in place, for example, waiting for landing approval at a station. What is the affect of false positive and false negative in setting throttle mix?